### DISCUSSION TITLE
I cant start up on mac

### DISCUSSION THREAD

**Discussion by doro-on** [Homebrew/discussions#85] [Everyday usage]
When i type the code, it says Password, but then i can't type anything, can someone help me?
![Captura de pantalla 2020-10-22 a la(s) 11 51 29](https://user-images.githubusercontent.com/71563237/96889496-05fe5480-145d-11eb-84bd-a7a4052c1d65.png)

**Comment by Rylan12** [comment]
This is not the right place to ask this question.

When you're prompted to enter your password in this way it will not show the characters as you type. You can still type the password and hit the enter key when you're done. It will register the keystrokes even though it doesn't show the characters.
